lib/trubl/base.rb

```diff
@@ -28,7 +28,9 @@ def missing_or_exception?(response)
           nil
         end
 
-        code && (400..600).include?(code)
+        is_missing_or_exception = code && (400..600).include?(code)
```

Comment:
This logic looks repeated in both base.rb and collection.rb. It may be worth abstracting to some sort of ResponseHandler class or something.